modules/local/junction_track.nf

```diff
@@ -0,0 +1,42 @@
+process JUNCTION_TRACK {
+    tag 'junction_track'
+    label 'process_low'
+
+    conda "conda-forge::python=3.8.3"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.8.3' :
+        'quay.io/biocontainers/python:3.8.3' }"
+
+    input:
+    tuple val(meta), path(sj)
+
+    output:
+    tuple val(meta), path('*.bed')    , emit: bed
+    path "versions.yml" , emit: versions
```

Comment:
could you align the emit statements, my OCD nerve is itching :)

Reply:
Done